templates/mod.html

```diff
@@ -75,7 +75,7 @@ <h1 title="{{ mod.name }}">
         <div class="{% if user %}col-md-2{% else %}col-md-4{% endif %}">
             <a  class="btn btn-block btn-lg btn-primary piwik_download" id="download-link-primary"
                 href="{{ url_for("mods.download", mod_id=mod.id, mod_name=mod.name, version=latest.friendly_version) }}">
-                Download</a>
+                Download {% if latest.id in size_versions and size_versions[latest.id] is not None %} ({{ size_versions[latest.id] }}) {% endif %}</a>
```

Comment:
`is not None` doesn't seem to work in Jinja2:
```
spacedock    | Traceback (most recent call last):
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/flask/app.py", line 2463, in __call__
spacedock    |     return self.wsgi_app(environ, start_response)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/flask/app.py", line 2449, in wsgi_app
spacedock    |     response = self.handle_exception(e)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/flask/app.py", line 1866, in handle_exception
spacedock    |     reraise(exc_type, exc_value, tb)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
spacedock    |     raise value
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
spacedock    |     response = self.full_dispatch_request()
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
spacedock    |     rv = self.handle_user_exception(e)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
spacedock    |     reraise(exc_type, exc_value, tb)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
spacedock    |     raise value
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
spacedock    |     rv = self.dispatch_request()
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
spacedock    |     return self.view_functions[rule.endpoint](**req.view_args)
spacedock    |   File "/opt/spacedock/KerbalStuff/common.py", line 73, in go
spacedock    |     ret = f(*args, **kw)
spacedock    |   File "/opt/spacedock/KerbalStuff/blueprints/mods.py", line 186, in mod
spacedock    |     'size_versions': size_versions
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/flask/templating.py", line 138, in render_template
spacedock    |     ctx.app.jinja_env.get_or_select_template(template_name_or_list),
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/environment.py", line 869, in get_or_select_template
spacedock    |     return self.get_template(template_name_or_list, parent, globals)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/environment.py", line 830, in get_template
spacedock    |     return self._load_template(name, self.make_globals(globals))
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/environment.py", line 804, in _load_template
spacedock    |     template = self.loader.load(self, name, globals)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/loaders.py", line 125, in load
spacedock    |     code = environment.compile(source, name, filename)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/environment.py", line 591, in compile
spacedock    |     self.handle_exception(exc_info, source_hint=source_hint)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
spacedock    |     reraise(exc_type, exc_value, tb)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
spacedock    |     raise value.with_traceback(tb)

spacedock    |   File "/opt/spacedock/templates/mod.html", line 78, in template
spacedock    |     Download {% if latest.id in size_versions and size_versions[latest.id] is not None %} ({{ size_versions[latest.id] }}) {% endif %}</a>

spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/environment.py", line 543, in _generate
spacedock    |     optimized=self.optimized)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/compiler.py", line 82, in generate
spacedock    |     generator.visit(node)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/visitor.py", line 38, in visit
spacedock    |     return f(node, *args, **kwargs)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/compiler.py", line 799, in visit_Template
spacedock    |     self.blockvisit(block.body, block_frame)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/compiler.py", line 378, in blockvisit
spacedock    |     self.visit(node, frame)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/visitor.py", line 38, in visit
spacedock    |     return f(node, *args, **kwargs)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/compiler.py", line 1157, in visit_If
spacedock    |     self.visit(node.test, if_frame)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/visitor.py", line 38, in visit
spacedock    |     return f(node, *args, **kwargs)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/compiler.py", line 70, in new_func
spacedock    |     return f(self, node, frame, **kwargs)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/compiler.py", line 1484, in visitor
spacedock    |     self.visit(node.right, frame)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/visitor.py", line 38, in visit
spacedock    |     return f(node, *args, **kwargs)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/compiler.py", line 70, in new_func
spacedock    |     return f(self, node, frame, **kwargs)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/compiler.py", line 1497, in visitor
spacedock    |     self.visit(node.node, frame)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/visitor.py", line 38, in visit
spacedock    |     return f(node, *args, **kwargs)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/compiler.py", line 70, in new_func
spacedock    |     return f(self, node, frame, **kwargs)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/compiler.py", line 1607, in visit_Test
spacedock    |     self.fail('no test named %r' % node.name, node.lineno)
spacedock    |   File "/venv/spacedock/lib/python3.6/site-packages/jinja2/compiler.py", line 315, in fail
spacedock    |     raise TemplateAssertionError(msg, lineno, self.name, self.filename)
spacedock    | jinja2.exceptions.TemplateAssertionError: no test named 'None'
```

Found something here: https://stackoverflow.com/a/30376144

Reply:
https://github.com/pallets/jinja/issues/520

use `is not none`, lower case.

```suggestion
                Download {% if latest.id in size_versions and size_versions[latest.id] is not none %} ({{ size_versions[latest.id] }}) {% endif %}</a>
```